### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tmteamz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,7 +182,9 @@ coin-hive <YOUR-MONERO-ADDRESS> --pool-host=xmr-eu1.nanopool.org --pool-port=144
 
 **Can I run this on Heroku?**
 
-Yes, but since Puppeteer requires some additional dependencies that aren't included on the Linux box that Heroku spins up for you, you need to go to your app's `Settings > Buildpacks`  first and add this url:
+No, It's violate the [Terms](https://www.heroku.com/policy/aup).
```
Comment (soyuka) [inline README.md]: it violates the TOS

File: README.md [hunk 2]
```diff
@@ -182,7 +182,9 @@ coin-hive <YOUR-MONERO-ADDRESS> --pool-host=xmr-eu1.nanopool.org --pool-port=144
...
+
+Take your own risk, and since Puppeteer requires some additional dependencies that aren't included on the Linux box that Heroku spins up for you, you need to go to your app's `Settings > Buildpacks`  first and add this url:
```
Comment (soyuka) [inline README.md]: Also, since

**Comment by tmteamz** [comment]
english is not my frist language. sorry

**Comment by soyuka** [comment]
Neither is it mine.

File: README.md [hunk 3]
```diff
@@ -182,7 +182,9 @@ coin-hive <YOUR-MONERO-ADDRESS> --pool-host=xmr-eu1.nanopool.org --pool-port=144
 
 **Can I run this on Heroku?**
 
-Yes, but since Puppeteer requires some additional dependencies that aren't included on the Linux box that Heroku spins up for you, you need to go to your app's `Settings > Buildpacks`  first and add this url:
+No, It violate the [Terms](https://www.heroku.com/policy/aup).
```
Comment (soyuka) [inline README.md]: No, it violates

File: README.md [hunk 4]
```diff
@@ -182,7 +182,9 @@ coin-hive <YOUR-MONERO-ADDRESS> --pool-host=xmr-eu1.nanopool.org --pool-port=144
...
+
+Take your own risk. Also, since Puppeteer requires some additional dependencies that aren't included on the Linux box that Heroku spins up for you, you need to go to your app's `Settings > Buildpacks`  first and add this url:
```
Comment (soyuka) [inline README.md]: Remove "Take your own risk"

**Comment by tmteamz** [comment]
DONE!

**Pull request merged**